src/components/input/index.jsx

```diff
@@ -0,0 +1,19 @@
+import { InputStyle } from "./style";
+
+export const Input = ({ placeholder, ...props }) => {
+
+    return (
+        <InputStyle 
+            {...props}
+            type='text'
+            placeholder={placeholder}
+            style={{
+                'border': 'none',
```

Comment:
Instead of use native style tag try to use the Material Component such as [TextField](https://mui.com/material-ui/react-text-field/) component. Another option is to style the component using the className since you have installed boostrap